extension/oidcextension/README.md

```diff
@@ -0,0 +1,36 @@
+# OIDC
+
+This extension enables bearer token authorization for GRPC receivers.
+
+The following settings are required:
+
+- `issuer_url`: The base URL for the Config provider.
+- `audience`: Audience of the token, used during the verification
+- `attribute` (default: `authorization`): The attribute (header name) to look for auth data
```

Comment:
If it has a default, it's not required :-)

Reply:
True lol. I just wanted to call out that setting, but its probably not that necessary